app/controllers/shopping_lists_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ShoppingListsController < ApplicationController
+  def index
+    @all_foods = current_user
+      .recipes
+      .includes([:recipe_foods])
+      .joins('INNER JOIN recipe_foods ON recipe_foods.recipe_id = recipes.id')
+      .joins('INNER JOIN foods ON recipe_foods.food_id = foods.id')
+      .select("foods.name as name,
+                foods.measurement_unit as measurement_unit,
+                sum(recipe_foods.quantity) as quantity,
```

Comment:
Please refactor this section of your code to use Active Records instead of directly using SQL queries.
It makes the code simple, and easy to understand. Thank you